components/containers/Dashboard/DataView.tsx

```diff
@@ -34,33 +34,37 @@ export const DataView = (props: DataViewProps): React.ReactElement => {
 
 const DataElement = (props: { template: FormDBConfigProperties }): React.ReactElement => {
   const { t, i18n } = useTranslation("admin-templates");
-  const [isExpanded, setIsExpanded] = useState(false);
   const { template } = props;
   const formID = template.formID;
   const router = useRouter();
 
-  const toggleExpanded = () => {
-    setIsExpanded(!isExpanded);
-  };
-
   const redirectToSettings = (formID: number) => {
     router.push({
       pathname: `/${i18n.language}/id/${formID}/settings`,
     });
   };
 
+  let title = "";
+  if (template.formConfig) {
```

Comment:
Wondering if we should set a fall back here in case internalTitleEn/Fr doesn't exist (the type def has it as possibly null) and leverage the `formConfig.form.titleEn/Fr` which we know can't be null.

Reply:
Good call. I'll add that in.